src/workspaces/dashboard/WorkspaceTags.test.ts

```diff
@@ -130,7 +130,7 @@ describe('WorkspaceTags', () => {
       Metrics: { captureEvent: mockCaptureEvent } as Partial<AjaxContract['Metrics']>,
       Workspaces: {
         // the tags select component still calls this
-        getTags: jest.fn().mockResolvedValue([initialTags]),
+        getTags: jest.fn().mockResolvedValue([]),
```

Comment:
I tried copying this example and was confused when it didn't actually work. The Ajax response for `getTags` is not an array of strings… it's an array of objects with tag and count. This test didn't actually need the Ajax response to return the initial tags because they were passed into the component being tested as initial values (the Ajax response just returns suggestions for the input select).